src/lesson/lesson9/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import TableOfUsers from './TableOfUsers'
+
+const USERS = [
+  {
+    id: 1,
+    firstName: 'David',
+    lastName: 'Dubliakov',
+    age: '19'
+  },
+  {
+    id: 2,
+    firstName: 'Dubliakov',
+    lastName: 'David',
+    age: '21'
+  },
+  {
+    id: 3,
+    firstName: 'Bohdan',
+    lastName: 'Semenov',
+    age: '19'
+  },
+  {
+    id: 4,
+    firstName: 'Semenov',
+    lastName: 'Bohdan',
+    age: '30'
+  },
+  {
+    id: 5,
+    firstName: 'Misha',
+    lastName: 'Kvochko',
+    age: '20'
+  },
+  {
+    id: 6,
+    firstName: 'Kvochko',
+    lastName: 'Misha',
+    age: '11'
+  },
+]
+
+const Main = () => {
+
+  const [users, setUsers] = useState(USERS)
```

Comment:
You have warning in terminal due to unused setUser hook. Just remove state and pass constant instead of state.
![image](https://user-images.githubusercontent.com/62145988/169048519-0fd8f9de-1e1a-419b-a3fa-42a4fac72a8a.png)

Reply:
fixed